[Docs] Init DDD cookbook (#196)
Sylius/Stack: docs/cookbook/ddd_architecture.md

# How to use in a DDD architecture

In a Domain-Driven Design architecture, there are no defined structures. 

> Here we present a lot of tips to help you to integrate the Sylius stack components in your clean architecture.

Reviewer: ```suggestion
Here, we present a lot of tips to help you integrate the Sylius Stack components in your clean architecture.
```

> In this cookbook we reuse the apip-ddd project from Mathias Arlaud and Robin Chalas for their excellent __Domain-driven design with API Platform 3__ talk.

Reviewer: ```suggestion
In this cookbook, we reuse Mathias Arlaud's and Robin Chalas's [apip-ddd project](https://github.com/mtarld/apip-ddd) presented during their excellent __Domain-driven design with API Platform 3__ talk.
```
